docs/Usage.md

```diff
@@ -276,6 +276,43 @@ The `customFormInputs` define the logic that translates abstract "Input Types" i
 
 The `tooltipDescriptions` allows an implementation of the `FormBuilder` that changes the tooltip descriptions that appear on hover over certain areas of the tool. The `add` popup appears when hovering over the plus buttons, the `cardObjectName` is the name of the back end name that appears in every card object input, the `cardDisplayName` allows rewriting the description of the display name tooltip, the `cardDescription` option allows overwriting the tooltip for the description, and the `cardInputType` allows setting a custom tooltip for the Input Type dropdown.
 
+### Deactivated Form Inputs
```

Comment:
Would you mind deleting lines 258 through 273 (https://github.com/ginkgobioworks/react-json-schema-form-builder/pull/69/files#diff-ec6d566f8253b138e4ea6289eac97191470b28549ad6c2e0cb86d4ac71ee3f58R258)? ("The full type definition..." through to the end of the `export type Mods` code block -- it's out of date and probably best if we just left it off)

Reply:
Done ✅